_layouts/page.html

```diff
@@ -133,6 +133,9 @@
       ga('send', 'pageview');
     </script>
 
+    <!-- Sandstorm stats collection -->
```

Comment:
This means we'd send events on all page views. That's the idea, right?

Just wanted to make sure.